src/libraries/System.Net.Mail/src/System/Net/Mail/SmtpClient.cs

```diff
@@ -829,7 +829,7 @@ public Task SendMailAsync(MailMessage message, CancellationToken cancellationTok
                     finally
                     {
                         if (e.Error != null) tcs.TrySetException(e.Error);
-                        else if (e.Cancelled) tcs.TrySetCanceled();
+                        else if (e.Cancelled) tcs.TrySetCanceled(cancellationToken);
```

Comment:
Did the analyzer warn on this?  This is a case where it's grabbing it from outside of the lambda.  We should revert this one.

Reply:
You're right. Reverted.
I know what happened. I didn't copy the generated roslyn-analyzers DLLs correctly to my .nuget package, and I was still getting that suggestion.
I built everything again, deleted the .nuget folder, restored packages, copied the generated DLLs, and this suggestion is not showing up anymore.